web/src/components/NewSecretModalButton/NewSecretModalButton.tsx

```diff
@@ -144,6 +144,7 @@ export const NewSecretModalButton: React.FC<NewSecretModalButtonProps> = ({
                       tooltipProps={{
                         dataTooltipId: 'secretDescriptionTextField'
                       }}
+                      maxLength={5000000}
```

Comment:
Thanks @rambhosale for the PR. Can you move this number to `web/src/Config.ts`, name it something like `SECRET_LIMIT_IN_BYTES` or something like that, and use it here?

Also it stead of `5000000`, let's make it `5_242_880`?

Reply:
Hi @tan-nhu I have extracted the length to the config file and also made a similar change in the secrets update section, that was missed last time

Reply:
Thank you for making the change. The UI code looks good. I'll sync with my team tomorrow to see if any backend is needed to support increasing this secret length. Will follow up.

Reply:
Confirmed there's no extra backend change needed to support this.